includes/block-patterns/course/templates/course-default.php

```diff
@@ -1,8 +1,10 @@
 <?php
 /**
- * Default pattern content.
+ * Deprecated Default pattern content.
+ * Remove after the course_outline_ai feature is released.
  *
  * @package sensei-lms
+ * @deprecated $$next-version$$
```

Comment:
Is there a reason for deprecating the old templates? Are we planning to delete the old files and use the `v2` ones, or will the code from `v2` go into the old template files? 🤔

Reply:
Yes, we will delete the old templates and keep the v2.